test/test.c

```diff
@@ -41,14 +41,29 @@ UBENCH(c, do_nothing) {
   UBENCH_DO_NOTHING(b);
 }
 
+UBENCH_EX(c, ex)
+{
+  int b[1024];
+  int i;
+  int sum;
+  memset(b, 0x0, sizeof(b));
+
+  UBENCH_KEEP_RUNNING()
```

Comment:
`UBENCH_KEEP_RUNNING` -> `UBENCH_DO_BENCHMARK` maybe?

Reply:
I'm fine with that!